### ISSUE TITLE
Usage of the underscore-db mixin in the browser.

### ISSUE THREAD

**Issue by jcloutz** [typicode/lowdb#141]
Is it possible to use the underscore-db mixin in the browser to ID based resources?

I am working on a prototype where the data will be read in (initialized) from a json file. Then throughout the demonstration local storage will be used as the database. The goal is to make it look like the app is talking to a server without having actually have one then be returned to it's initial state by clearing local storage.

**Comment by typicode** [comment]
Hi @jcloutz,

Yes, underscore-db supports the browser and Node. So you can either require it and bundle your code with webpack or browserify for the browser or use the CDN:

```html
<script src="https://unpkg.com/lodash@4/lodash.min.js"></script>
<script src="https://unpkg.com/underscore-db@0.12.1/dist/underscore-db.min.js"></script>
<script src="https://unpkg.com/lowdb/dist/lowdb.min.js"></script>
<script>
  var db = low('db')
  db.insert({ foo: bar }).value()
</script>
```

There's an example with localStorage here https://jsfiddle.net/typicode/4kd7xxbu/ if you need.

**Comment by jcloutz** [comment]
@typicode thank you for your reply. I do seem to be having a problem getting it to work though. It's throwing the following error `Uncaught TypeError: db.insert`.

When checking in the console `_.insert()` is accessible to the global lodash instance. However, `db.insert()` and `db._.insert()` do not exist in the lowdb instance for some reason. Any ideas?

**Comment by jcloutz** [comment]
I figured it out. If i do the following I get the insert function on collections.

```js
db._.mixin(_)

db.get('collection').insert({foo: bar}).value()
```

After getting it working I realized that it was using a uuid based id instead of an incrementing id so I ended up writing a mixin similar to json-server.

```js
db._.mixin({
	insert: function(col, doc) {
		let id = col.length > 0 ? _.maxBy(col, 'id')['id'] : 0
		doc.id = ++id
		col.push(doc)
		return col
	}
})
```
Probably not very efficient on large collections but for my current purposes it will work.

Thank you for your quick response, this is an awesome project!
